lib/Converter/FunctionConverter.cpp

```diff
@@ -126,6 +127,12 @@ void FunctionConverter::convertOutputs(Node &node) {
       if (user == conversionVal.getNode()) {
         continue;
       }
+      // Log the change of node input(operand).
+      if (node.getParent()) {
+        node.getParent()->getLogContext().logNodeInputChange(
+            user, index++, use.get()->getNode(), conversionVal.getNode());
```

Comment:
is conversionVal.getResNo() the same as index always? Maybe don't need the new counter.